symposion/reviews/views.py

```diff
@@ -112,9 +138,9 @@ def review_section(request, section_slug, assigned=False):
         assignments = ReviewAssignment.objects.filter(user=request.user).values_list("proposal__id")
         queryset = queryset.filter(id__in=assignments)
 
-    queryset = queryset.select_related("result").select_subclasses()
+    queryset = queryset.select_related("result", "kind", "speaker__user", "tags").select_subclasses()
 
-    proposals = proposals_generator(request, queryset)
+    proposals = list(proposals_generator(request, queryset))
```

Comment:
Two of three places that call `proposals_generator` use it as a list. The third is similar to the others so I think that it could be used that way as well. Perhaps the function can return a list?